src/main/resources/application-dev.yml

```diff
@@ -6,7 +6,7 @@ spring:
 
   flyway:
     enabled: true
-    locations: classpath:db/migration,classpath:db/test/migration
+    locations: classpath:db/migration/general,classpath:db/migration/{vendor},classpath:db/test/migration
```

Comment:
unfortunately migration for h2 and postgres has diverged so need scripts in separate directories